hotel_roommatik/models/inherited_hotel_folio.py

```diff
@@ -2,15 +2,36 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from odoo import api, fields, models, _
-import xml.etree.cElementTree as ET
+import json
 
 
-class Inherited_hotel_folio(models.Model):
+class hotel_folio(models.Model):
 
     _inherit = 'hotel.folio'
 
+    @api.model
+    def rm_get_reservation(self, Code=False):
+        folio_res = self.env['hotel.folio'].search([('id', '=', Code)])
+        json_response = dict()
+        if len(folio_res) > 0:
```

Comment:
Here can use "any" instead of "len".
any(["", {}]) -> False
len(["", {}]) -> 2

Reply:
I changed it